docs/ops/wslashdelay.toml

```diff
@@ -0,0 +1,63 @@
+["W/D.FBK"]
+prototype = "W/D.FBK level"
+short = "amount of feedback from read head to write head (s16V)"
+
+["W/D.MIX"]
+prototype = "W/D.MIX fade"
+short = "fade from dry to delayed signal"
+
+["W/D.LPASS"]
```

Comment:
Could `LPASS` be changed to `FILTER` or `FILT` so it aligns with crow? The specific reason is I've got plans to make the filter more flexible than lowpass only in the future. Using `FILTER` is thus open to changes in the future.